core/src/main/java/com/linecorp/armeria/common/util/SystemInfo.java

```diff
@@ -109,6 +111,21 @@
                 JETTY_ALPN_OPTIONAL_OR_AVAILABLE = false;
             }
         }
+
+        try {
+            String osName = Ascii.toUpperCase(System.getProperty("os.name", ""));
+            if (osName.startsWith("WINDOWS")) {
+                osType = osType().WINDOWS;
+            }
+            if (osName.startsWith("LINUX")) {
+                osType = osType().LINUX;
+            }
+            if (osName.startsWith("MAC")) {
+                osType = osType().MAC;
+            }
```

Comment:
We should add `osType = OsType.OTHERS;` here

Reply:
Yes. I removed the try catch statement and used the if else clause :)